packages/calypso-router/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.7.0
```

Comment:
Should this be `0.1.0` as per semver? Or maybe `1.0.0`? Feels weird to start from `0.7.0`.

Reply:
`0.7.0` indeed but what does `page.js` have to do with the first WordPress version?

Reply:
It's a good initial version if you want to set up your package for success 🙂 But seriously, it's a WordPress joke, we'll soon promote it to 1.0.0 anyway.

Reply:
And WordPress is an example that those jokes live forever 😉 Leaving it to your judgment though.